RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -84,11 +99,46 @@
     e.toModify(pfPileUpIso, enable = cms.bool(False))
     e.toModify(pfNoPileUp, enable = cms.bool(False))
     e.toModify(pfPileUp, enable = cms.bool(False))
-    
 
+#
+# for MLPF
 from Configuration.ProcessModifiers.mlpf_cff import mlpf
 from RecoParticleFlow.PFProducer.mlpfProducer_cfi import mlpfProducer
 
 _mlpfTask = cms.Task(mlpfProducer, particleFlowRecoTask.copy())
 
 mlpf.toReplaceWith(particleFlowRecoTask, _mlpfTask)
+
+#
+# switch from pfTICL to simPF
+def _findIndicesByModule(process,name):
+    ret = []
+    if getattr(process,'particleFlowBlock', None):
+        for i, pset in enumerate(process.particleFlowBlock.elementImporters):
+            if pset.importerName.value() == name:
+                ret.append(i)
+    return ret
+
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+def replaceTICLwithSimPF(process):
+    if getattr(process,'particleFlowTmp', None):
```

Comment:
`hasattr(process,'particleFlowTmp')` is slightly simpler

Reply:
If I don't hear any other suggestions, i will make this change soon.

Reply:
Done (including two other lines).